article_maker.cc

```diff
@@ -642,19 +665,56 @@ void ArticleRequest::bodyFinished()
 
         closePrevSpan = true;
 
-        head += string( R"(<div class="gddictname" onclick="gdExpandArticle(')" ) + dictId + "\');"
```

Comment:
I had to partially refactor this because this was almost unreadable and unmaintainable. I used QString because it has the `arg()` method that can be used to easily mimic `libfmt`.